packages/webapp/src/routes/developer/confirm-off-site-gain.js

```diff
@@ -0,0 +1,87 @@
+import constants from '../../utils/constants.js'
+import { deleteBlobFromContainers } from '../../utils/azure-storage.js'
+import { logger } from 'defra-logging-facade'
```

Comment:
log

Reply:
We added those logs for tracing purpose on POC deployment as we were not getting data after the deployment & will remove if its working fine after deployment